Extension/src/LanguageServer/extension.ts

```diff
@@ -584,6 +584,7 @@ export function registerCommands(): void {
     disposables.push(vscode.commands.registerCommand('C_Cpp.ResumeParsing', onResumeParsing));
     disposables.push(vscode.commands.registerCommand('C_Cpp.ShowParsingCommands', onShowParsingCommands));
     disposables.push(vscode.commands.registerCommand('C_Cpp.TakeSurvey', onTakeSurvey));
+    disposables.push(vscode.commands.registerCommand('cpptools.activeConfig', onGetActiveConfig));
```

Comment:
Should this be activeConfigName to be more explicit and/or if we want to expose the actual config or an index later?